### DISCUSSION TITLE
Image Prompt Pyra canny

### DISCUSSION THREAD

**Discussion by RedPrince0** [lllyasviel/Fooocus#3800] [Q&A] [answered]
I was using pyra canny in my images it works perfectly with thw base models, juggernaut, anime pencil, but im enjoying using pony model and the pyra canny is complete being ignored , resulting in not using the poses of my images from image prompt, any idea how to fix it or it actually doesnt work with pony v6?

**Comment by IPv6** [comment, marked as answer]
Unfortunately it actually doesn't work with pony v6 // Pyra canny trained for checkpoints that are similar to SDXL, and Pony is checkpoint that is trained with different assumptions. So many SDXL-centered control nets works differently with pony.

There is no solution, ideally control net should be retrained for pony

**Reply by RedPrince0** [reply]
So as long as it is a sdxl model the pyra canny will work?

**Reply by IPv6** [reply]
Control Nets will work with any SDXL model, but quality results can be archived with models that are similar to vanilla SDXL.
Pony was trained with many differences, so usual control nets are less effective with it
